src/components/Tooltips/TooltipWithSubtitle.tsx

```diff
@@ -1,5 +1,5 @@
 import { Tooltip, TooltipProps } from "antd";
-import React, { ReactElement, ReactNode, useRef } from "react";
+import React, { ReactElement, ReactNode } from "react";
```

Comment:
Removed the div ref because it introduced an extra level of nesting that could cause wrapped components to not behave correctly.